### PR TITLE
ci(Mergify): configuration update

### PR DESCRIPTION
**Pull request by Borda**
Each PR has to be approved by at least one Grid.ai member...
Need to be checked when it is merged to master if it works correctly
see: https://docs.mergify.com/actions/post_check/

cc @carmocca @akihironitta @borda

### CODE REVIEW

**Review by akihironitta — APPROVED** [review]

File: .github/mergify.yml [hunk 1]
```diff
@@ -68,3 +68,28 @@ pull_request_rules:
...
+        - approved-reviews-by=ethanwharris
+        - approved-reviews-by=kaushikb11
+        - approved-reviews-by=rohitgr7
+        - approved-reviews-by=tchaton
+        - approved-reviews-by=williamFalcon
+    actions:
+      post_check:
+        title: |
+          {% if check_succeed %}
+          This PR has Gris.ai approval.
```
Comment (kaushikb11) [inline .github/mergify.yml:88]:
```suggestion
          This PR has Grid.ai approval.
```

File: .github/mergify.yml [hunk 2]
```diff
@@ -68,3 +68,28 @@ pull_request_rules:
...
+          {% else %}
+          This PR is waiting approval from a Grid.ai person.
+          {% endif %}
+        summary: |
+          {% if not check_succeed %}
+          Each pull request has to be approved at least by one Grid.ai employee.
```
Comment (kaushikb11) [inline .github/mergify.yml:94]:
```suggestion
          Each pull request must be approved by at least one Grid.ai team member.
```

**Review by kaushikb11 — APPROVED** [review]

File: .github/mergify.yml [hunk 3]
```diff
@@ -68,3 +68,28 @@ pull_request_rules:
...
+        - approved-reviews-by=rohitgr7
+        - approved-reviews-by=tchaton
+        - approved-reviews-by=williamFalcon
+    actions:
+      post_check:
+        title: |
+          {% if check_succeed %}
+          This PR has Grid.ai approval.
+          {% else %}
+          This PR is waiting approval from a Grid.ai person.
```
Comment (kaushikb11) [inline .github/mergify.yml:90]:
```suggestion
          This PR is waiting for approval from a Grid.ai team member.
```

File: .github/mergify.yml [hunk 4]
```diff
@@ -68,3 +68,28 @@ pull_request_rules:
...
+      - or:
+        - approved-reviews-by=awaelchli
+        - approved-reviews-by=Borda
+        - approved-reviews-by=carmocca
+        - approved-reviews-by=edenlightning
+        - approved-reviews-by=ethanwharris
+        - approved-reviews-by=kaushikb11
+        - approved-reviews-by=rohitgr7
+        - approved-reviews-by=tchaton
+        - approved-reviews-by=williamFalcon
```
Comment (carmocca) [inline .github/mergify.yml:83]:
```suggestion
        - approved-reviews-by=williamFalcon
        - approved-reviews-by=krshrimali
```

File: .github/mergify.yml [hunk 5]
```diff
@@ -68,3 +68,28 @@ pull_request_rules:
       request_reviews:
         teams:
           - "@PyTorchLightning/core-lightning"
+
+  - name: Grid.ai approval
```
Comment (carmocca) [inline .github/mergify.yml:72]:
```suggestion
  - name: Grid.ai employee approval
```

**Review by carmocca — APPROVED** [review]
Nice!

File: .github/mergify.yml (hunk 4 above)
Comment (rohitgr7) [reply .github/mergify.yml:83]: and @otaj

Comment (Borda) [reply .github/mergify.yml:83]: let's add Ota later when he gets familiar with code-base

File: .github/mergify.yml [hunk 6]
```diff
@@ -68,3 +68,29 @@ pull_request_rules:
...
+        - approved-reviews-by=awaelchli
+        - approved-reviews-by=Borda
+        - approved-reviews-by=carmocca
+        - approved-reviews-by=edenlightning
+        - approved-reviews-by=ethanwharris
+        - approved-reviews-by=kaushikb11
+        - approved-reviews-by=rohitgr7
+        - approved-reviews-by=tchaton
+        - approved-reviews-by=williamFalcon
+        - approved-reviews-by=krshrimali
```
Comment (Borda) [inline .github/mergify.yml:84]:
```suggestion
        - approved-reviews-by=krshrimali
        - approved-reviews-by=rohitgr7
        - approved-reviews-by=tchaton
        - approved-reviews-by=williamFalcon
```

**Review by justusschock — APPROVED** [review]

**Pull request merged**